sort.py

```diff
@@ -53,6 +56,23 @@ def sort():
         blocks = [''.join(sorted(block, key=lambda s: s.lower())) for block in blocks]
         sorted_file.write(''.join(blocks))
     
+        with open(f'empresas.json', 'w+', encoding='UTF-8') as f:
+        for block in blocks:

```

Comment:
Hi dubaid, thanks for your contribuition. Please fix python identation error here.